src/abc_atlas_access/abc_atlas_cache/abc_project_cache.py

```diff
@@ -0,0 +1,495 @@
+        self,
+        cache: Union[S3CloudCache, LocalCache],
+        skip_version_check: bool = False,
+        local: bool = False
+    ):
+        self.cache = cache
+        self.skip_version_check = skip_version_check
+        self._local = local
+        self.load_manifest()
+        self._ui_class_name = self.__class__.__name__
+        self.logger = logging.getLogger(self._ui_class_name)
+
+
+    @property
+    def manifest(self):
+        if self.cache is None:
+            api_name = type(self.fetch_api).__name__
+            raise NotImplementedError(f"A {type(self).__name__} "
+                                      f"based on {api_name} "
+                                      "does not have an accessible manifest "
+                                      "property")
+        return self.cache.manifest
+
+    @property
+    def latest_manifest_file(self) -> str:
+        """
+        Return the name of the most up to date data manifest
+        corresponding to this dataset, checking in the cloud
+        if this is a cloud-backed cache.
+        """
+        return self.cache.latest_manifest_file
+
+    def load_manifest(self, manifest_name: Optional[str] = None):
+        """
+        Load the specified manifest file into the CloudCache
+
+        Parameters
+        ----------
+        manifest_name: Optional[str]
+            Name of manifest file to load. If None, load latest
+            (default: None)
+        """
+        if manifest_name is None:
+            self.cache.load_last_manifest()
+        else:
+            self.cache.load_manifest(manifest_name)
+
+        if not self.skip_version_check:
+            version_check(
+                self.cache.version,
+                cmin=self.MANIFEST_COMPATIBILITY[0],
+                cmax=self.MANIFEST_COMPATIBILITY[1])
+
+    def construct_local_manifest(self) -> None:
+        """
+        Construct the local file mapping path to file hash. (You will see a
+        warning if the cache thinks that you need to run this method).
+        """
+        self.cache.construct_local_manifest()
+
+    @classmethod
+    def from_s3_cache(
+        cls,
+        cache_dir: Union[str, Path]
+    ) -> "ProjectCloudApiBase":
+        """instantiates this object with a connection to an s3 bucket and/or
+        a local cache related to that bucket.
+
+        Parameters
+        ----------
+        cache_dir: str or pathlib.Path
+            Path to the directory where data will be stored on the local system
+
+        Returns
+        -------
+        BehaviorProjectCloudApi instance
+
+        """
+        cache = S3CloudCache(cache_dir=cache_dir,
+                             bucket_name=cls._bucket_name,
+                             ui_class_name=cls.__class__.__name__)
+        return cls(cache)
+
+    @classmethod
+    def from_local_cache(
+        cls,
+        cache_dir: Union[str, Path],
+    ) -> "ProjectCloudApiBase":
+        """instantiates this object with a local cache.
```

Comment:
Can you be a bit more explicit about what you mean by "local cache," e.g. "you will have access to files that have already been downloaded, but will not be able to download new files."